content/docs/tasks/traffic-management/egress-gateway/index.md

```diff
@@ -126,23 +125,25 @@ First direct HTTP traffic without TLS origination
     apiVersion: networking.istio.io/v1alpha3
     kind: DestinationRule
     metadata:
-      name: set-sni-for-egress-gateway
+      name: egressgateway-for-cnn
     spec:
       host: istio-egressgateway.istio-system.svc.cluster.local
-      trafficPolicy:
-        loadBalancer:
-          simple: ROUND_ROBIN
-        portLevelSettings:
-        - port:
-            number: 80
-          tls:
-            mode: MUTUAL
-            clientCertificate: /etc/certs/cert-chain.pem
-            privateKey: /etc/certs/key.pem
-            caCertificates: /etc/certs/root-cert.pem
-            subjectAltNames:
-            - spiffe://cluster.local/ns/istio-system/sa/istio-egressgateway-service-account
-            sni: edition.cnn.com
+      subsets:
+      - name: cnn
+        trafficPolicy:
+          loadBalancer:
+            simple: ROUND_ROBIN
+          portLevelSettings:
+          - port:
+              number: 80
+            tls:
+              mode: MUTUAL
+              clientCertificate: /etc/certs/cert-chain.pem
+              privateKey: /etc/certs/key.pem
+              caCertificates: /etc/certs/root-cert.pem
+              subjectAltNames:
+              - spiffe://cluster.local/ns/istio-system/sa/istio-egressgateway-service-account
```

Comment:
with the latest change to master, you can specify mode: ISTIO_MUTUAL and sni: edition.cnn.com.. omit the rest.

Reply:
Cool, will update it.

Reply:
@rshriram done.